apple-pie/styles/style.css

```diff
@@ -0,0 +1,62 @@
+* {
+
+font-family:sans-serif;
+font-weight: 200;
+margin-bottom: 20px;
```

Comment:
To get rid of the white border around the image you just need to add this line `margin: 0` here at the universal selecter `*`:

```
* {
  font-family: sans-serif;
  font-weight: 200;
  margin-bottom: 20px;
  margin: 0
}
```